tools/make_patch.c

```diff
@@ -0,0 +1,944 @@
+#include <stdlib.h>
+#include <stdio.h>
+#include <stdint.h>
+#include <errno.h>
+#include <string.h>
+#include <ctype.h>
+
+struct symbol {
+	struct symbol *next;
+	unsigned int value;
+	char name[];
+};
+
+struct patch {
+	unsigned int offset;
+	unsigned int size;
+};
+
+struct buffer {
+	size_t size;
+	char buffer[];
+};
+
+struct asmfile {
+	struct asmfile *next;
+	struct symbol *symbols;
+	char name[];
+};
+
+struct asmfile *asmfiles = NULL;
+
+void free_buffer(void *buf)
+{
```

Comment:
The other tools put open braces on the same line as functions, like how the structs above are written.

Reply:
Resolved in https://github.com/pret/pokecrystal/pull/882/commits/384aa73bfedc1c68efa000ab748fa178abf4f159